apps/atomy-q/API/database/factories/ApprovalFactory.php

```
@@ -0,0 +1,88 @@
+use App\Models\User;
+use App\Models\Tenant;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+final class ApprovalFactory extends Factory
+{
+    protected $model = Approval::class;
+
+    public function definition(): array
+    {
+        return [
+            'tenant_id' => Tenant::factory(),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\Tenant' in method 'definition'. (undefined)

(StaticAccess)